src/main/java/com/vinyl/repository/UserRepository.java

```diff
@@ -8,5 +10,6 @@
 @Repository("userRepository")
 public interface UserRepository extends CrudRepository<User, Integer> {
 
-	User findByEmail(String email);
+	Optional<User> findByEmail(String email);
+	User findByPass(String password);
```

Comment:
please delete this method as it is not used

Reply:
fixed